### ISSUE TITLE
python2.7: can't load library 'libpthread.so.1'

### ISSUE THREAD

**Issue by TheJeremyP** [Optware/Optware-ng#217]
Right after bootstrapping and installing python, trying to run python gives me.

`/opt/bin/python2.7: can't load library 'libpthread.so.1'`

More details: This is on an R7000 running AdvancedTomato.
There's a libpthread.so in libc-dev, but that's a symlink to libpthread.so.0 that doesn't seem to exist anywhere and trying to install libc-dev gives a dependency error. 

```
* satisfy_dependencies_for: Cannot satisfy the following dependencies for libc-dev:
*       *
* opkg_install_cmd: Cannot install package libc-dev.
```
Which I think is caused by the empty space between the commas in the control file:
`Depends: uclibc-opt,  , libstdc++`

**Comment by alllexx88** [comment]
Fixed with 5442472fa83a61f0bccfc49f97fa9d62e749c02d
Please upgrade `libdb` package:
```
ipkg update
ipkg upgrade libdb
```

**Comment by TheJeremyP** [comment]
Thanks for the quick fix. Might also want to take a look at the python and python27 packages. The symlinks for idle, pydoc, and smtpd.py in the python package link to idle2.7, pydoc2.7, and smtpd2.7.py, but the python27 package installs as idle-2.7, pydoc-2.7, smtpd-2.7. One set has hyphens, the other doesn't.

**Comment by alllexx88** [comment]
Thanks, symlinks fixed too: 341509d93d7873e556676938a3d4ae1505a0bbc7
